README.md

```diff
@@ -90,6 +85,10 @@ psql osmose_frontend -f tools/database/schema.sql
 
 Check data base parameter into `tools/utils.py`.
 
+### Generate markers
+```
+cd tools && ./make-markers.py
```

Comment:
We need in fact to complete the command, as it is not sufficient to update markers.

I also run:
```(cd web && npm run build) && touch osmose.wsgi

Reply:
I'm not sure I understand. `npm run build` is covered in the Javascript section. I can merge both sections if that's what you meant ?

Reply:
make-maker create new images, and need to be bundled by  `npm run build`

Reply:
[Right](https://github.com/osm-fr/osmose-frontend/blob/master/web/webpack.config.js#L47-L48), nice catch !

I moved asset generation after `make-markers`, but didn't merge both sections as `webpack` also build JS and CSS, that are unrelated to markers I believe.